src/controllers/tx-status.controller.ts

```diff
@@ -16,7 +18,9 @@ export class TxStatusController {
       @inject(ServicesBindings.PEGIN_STATUS_SERVICE)
       protected peginStatusService: PeginStatusService,
       @inject(ServicesBindings.PEGOUT_STATUS_SERVICE)
-      protected pegoutStatusService: PegoutStatusService
+      protected pegoutStatusService: PegoutStatusService,
+      @inject(ServicesBindings.REGISTER_FLYOVER_SERVICE)
+      protected registerFlyoverService: RegisterFlyoverService,
```

Comment:
Better name to this service is PegoutFlyoverService, soon we will create the PeginFlyoverService, it can be about the feature and not about the operation.